### PR TITLE
glib 2.74.4

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/glib.rb [hunk 1]
```diff
@@ -17,15 +17,19 @@ class Glib < Formula
     sha256 x86_64_linux:   "d896a5a0f3b720819fd86a85a95c7385b50212918ef9d7db8adb5750e2c2ed99"
   end
 
+  depends_on "gettext" => :build
```
Comment (cho-m) [inline Formula/glib.rb:20]: If available at build time, will be used to generate MO files. Removing `gettext` entirely results in not generating/installing `share/locale/*/LC_MESSAGES/glib20.mo`.

Runtime dependency is for systems that need libintl like macOS. GNU/Linux uses glibc instead.

**Comment by cho-m** [comment]
Only need to update some lost indirect linkage to `libxml2`:
```
brew linkage --cached --test --strict fuse-emulator
brew linkage --cached --test --strict open-babel
brew linkage --cached --test --strict rrdtool
```

**Comment by cho-m** [comment]
Want to get `qt` bottle (#118320) to avoid Linux CI trying to build `qt` bottle during dependent tests. Should help reduce CI time and clean up Linux logs before trying another run.

**Comment by fxcoudert** [comment]
For a minor update of a formula where we don't expect any ABI breakage, I would recommend we use `skip recursive dependents` from now on. It takes too much time testing (and triggering various test flakiness).

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
## 12-arm

```
brew test --retry --verbose vice
```

## 12

```
brew install --only-dependencies chromaprint
```

## linux

```
brew test --retry --verbose openslide
```

**Comment by chenrui333** [comment]
> ## 12-arm
> ```
> brew test --retry --verbose vice
> ```

should be addressed #118935

**Comment by chenrui333** [comment]
> ## linux
> ```
> brew test --retry --verbose openslide
> ```

addressed in #118939

**Comment by chenrui333** [comment]
> ## 12
> ```
> brew install --only-dependencies chromaprint
> ```

just some networking failure which can be ignored.

**Pull request closed without merging**